docs/data-sources/cdn_domain_statistics.md

```diff
@@ -0,0 +1,101 @@
+---
+subcategory: Content Delivery Network (CDN)
+---
+
+# huaweicloud_cdn_domain_statistics
+
+Use this data source to get the statistics of CDN domain.
+
+## Example Usage
+
+```hcl
+variable "domain_name" {}
+
+data "huaweicloud_cdn_domain_statistics" "test" {
+  domain_name = "terraform.test.huaweicloud.com"
+  action      = "location_detail"
+  start_time  = 1662019200000
+  end_time    = 1662021000000
+  stat_type   = "req_num"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `domain_name` - (Required, String) Specifies the domain name list. Domain names are separated by commas (,),
+for example, www.test1.com,www.test2.com. The value all indicates that all domain names under your account are queried.
+
+* `action` - (Required, String) Specifies the action name. Possible values are: **location_summary** and **location_detail**.
```

Comment:
we can genetate the validation func with **enum** in yaml

Reply:
fixed